tensorflow/lite/kernels/CMakeLists.txt

```diff
@@ -17,7 +17,26 @@ find_package(googletest REQUIRED)
 find_package(nsync REQUIRED)
 find_package(re2 REQUIRED)
 
+# Generate the mutable schema_generated.h header for tests.
+set(FLATC_PATH ${CMAKE_BINARY_DIR}/flatbuffers-flatc/bin/flatc)
+set(FLATC_OPTION
+  -c
+  --no-union-value-namespacing
+  --gen-object-api
+  --gen-mutable
+)
+set(SCHEMA_FILE ${TFLITE_SOURCE_DIR}/schema/schema.fbs)
+set(SCHEMA_GENERATED_ROOT ${CMAKE_CURRENT_BINARY_DIR}/schema)
+set(SCHEMA_GENERATED_PATH ${SCHEMA_GENERATED_ROOT}/tensorflow/lite/schema/mutable)
+set(SCHEMA_GENERATED_FILE ${SCHEMA_GENERATED_PATH}/schema_generated.h)
+add_custom_command(
```

Comment:
Can't we use build_flatbuffers() in https://github.com/google/flatbuffers/blob/master/CMake/BuildFlatBuffers.cmake ?

Reply:
> Can't we use build_flatbuffers() in https://github.com/google/flatbuffers/blob/master/CMake/BuildFlatBuffers.cmake ?

I need to check if this is available using ExternalProject_Add() flow.

Reply:
@terryheo 
Here is the commit to use build_flatbuffers().
https://github.com/tensorflow/tensorflow/pull/47754/commits/809468d4407beaf80064b90f8c6d945de6394070